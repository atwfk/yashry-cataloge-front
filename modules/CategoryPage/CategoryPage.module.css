.div {
  @apply text-2xl bg-red-400;
}
