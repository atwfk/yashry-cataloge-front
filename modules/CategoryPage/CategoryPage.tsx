import React from "react";
import type { FC, ReactElement } from "react";
import Products from "@modules/shared/components/Orgnisms/Products/Products";
import { IData } from "@modules/shared/types/IData";

const CategoryPage: FC<{ data: IData.IProduct[] }> = ({
  data,
}): ReactElement => {
  return (
    <div>
      <Products products={data} />
    </div>
  );
};

export default CategoryPage;
