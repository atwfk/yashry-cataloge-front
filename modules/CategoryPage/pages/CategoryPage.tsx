import React from "react";
import type { FC, ReactElement } from "react";
import { IData } from "@modules/shared/types/IData";
import Categories from "../components/Categories/Categories";
import ProductFilters from "../components/ProductFilters/ProductFilters";

const CategoryPage: FC<IData.IProps> = ({ data }): ReactElement => {
  return (
    <>
      <Categories />
      <ProductFilters data={data} />
    </>
  );
};

export default CategoryPage;
