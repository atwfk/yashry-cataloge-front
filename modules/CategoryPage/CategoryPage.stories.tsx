import React from "react";
import type { Story, Meta } from "@storybook/react";
import CategoryPage from "./CategoryPage";

export default {
  title: "TS/CategoryPage",
  component: CategoryPage,
} as Meta;

const Template: Story = (args) => <CategoryPage {...args} />;

export const textInput = Template.bind({});
