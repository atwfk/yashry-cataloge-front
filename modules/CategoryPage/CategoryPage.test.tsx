import React from "react";
import renderer from "react-test-renderer";
import CategoryPage from "./CategoryPage";

describe("CategoryPage", () => {
  it("should render Category Page text", () => {
    const tree = renderer.create(<CategoryPage />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
