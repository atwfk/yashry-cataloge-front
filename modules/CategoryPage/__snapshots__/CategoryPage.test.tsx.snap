// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryPage should render Category Page text 1`] = `
<div
  className="div"
>
  Category Page
</div>
`;
